src/view/View.js

```diff
@@ -1391,12 +1391,7 @@ new function() { // Injection scope for event handling on the browser
             if (mouse.down && inView || mouse.up && downPoint) {
                 emitMouseEvents(this, hitItem, type, event, point, downPoint);
                 if (mouse.down) {
-                    // See if we're clicking again on the same item, within the
-                    // double-click time. Firefox uses 300ms as the max time
-                    // difference:
-                    dblClick = hitItem === clickItem
-                        && (Date.now() - clickTime < 300);
-                    downItem = clickItem = hitItem;
```

Comment:
@sasensi this removes the check for clicks on the same item entirely... Are you sure we want this? Is it fine to do so because double-clicks are only considered within 3 pixels distance? But what if the first click was on a different item?

Reply:
No, you're right, I hadn't noticed that. But even if I don't manage to make it fail practically, from a theoretical point of view, it would be better to also check that item is the same for both clicks.
In new version, I also ensure that triple clicking an item don't produce 2 double click events.